ledger/ledger-api-bench-tool/src/test/suite/scala/com/daml/ledger/api/benchtool/metrics/LatencyMetricSpec.scala

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool
+
+import com.daml.ledger.api.benchtool.metrics.LatencyMetric
+import com.daml.ledger.api.benchtool.metrics.LatencyMetric.MaxLatency
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import scala.util.chaining._
+
+import java.time.Duration
+
+class LatencyMetricSpec extends AnyWordSpec with Matchers {
+  private val dummyPeriod = Duration.ofSeconds(1L)
+
+  LatencyMetric.getClass.getSimpleName should {
+    "compute correct values on updates" in {
+      LatencyMetric
+        .empty(maxLatencyObjectiveMillis = 0L)
+        .tap(_.finalValue(dummyPeriod) shouldBe LatencyMetric.Value(0L))
```

Comment:
Let's test `periodicValue()` as well for completeness.

Reply:
Sure